modules/computer.c

```diff
@@ -294,7 +294,8 @@ void scan_dev(gboolean reload)
        { N_("LLDB"), "lldb --version", "(?<=lldb version )(\\d\\.?)+", TRUE },
        { N_("strace"), "strace -V", "\\d+\\.\\d+\\.?\\d*", TRUE },
        { N_("valgrind"), "valgrind --version", "\\d+\\.\\d+\\.\\S+", TRUE },
-       { N_("QMake"), "qmake --version", "\\d+\\.\\S+", TRUE},
+       { N_("QMake (Qt 5)"), "qmake-qt5 --version", "\\d+\\.\\S+", TRUE},
```

Comment:
`qmake` binary (or symlink) is not always installed, and there can be more than one present.

Reply:
where are qt4 related here? some 3thr party system only runs in qt4

Reply:
Thanks for pointing this out. I remember qt4 being unsupported for five years, but it looks like [at least stable debian has it](https://packages.debian.org/buster/qt4-qmake), thus it should be supported here as well.

Reply:
it does not matter is are supproted or not.. currently there are complete desktop build around qt4 you know it? trinity desktop!  and some NASA related programs only runs under qt4 and python2 you know it?